src/configuration.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { ResponseType } from './responseType';
+
+import { RequestOptions } from './index';
+
+export interface RequestType {
+  options?: RequestOptions;
+}
+
+export interface RequestTypeWithBody<T = any> extends RequestType {
+  data: T;
+}
```

Comment:
what if we go with something like this:

```suggestion
export interface RequestType<T = never> {
  options?: RequestOptions;
  data?: T;
}
```

This way. we should be able to use `RequestType` when we don't need a body and `RequestType<Data>` when we do. What do you think?